src/components/AuthorCardEditing.vue

```diff
@@ -29,6 +35,22 @@
         <div class="row">
             <h3 class="subquestion">
                 The person's last names, split into parts
+                <q-icon
+                    name="ion-information-circle-outline"
+                    size="24px"
+                    color="primary"
+                    v-on:click="showHelpDialogLastName = true"
+                    style="cursor:pointer;"
+                />
+                <SchemaGuideLink anchor="#definitionspersonname-particle" />
+                <SchemaGuideLink
+                    anchor="#definitionspersonname-suffix"
+                    class="q-pt-sm"
+                />
+                <SchemaGuideLink
+                    anchor="#definitionspersonfamily-names"
+                    class="q-pt-sm"
+                />
```

Comment:
```suggestion
```
The SchemaGuideLink was removed. I don't know why it did not raise an error in either build or lint.